index.rst

```diff
@@ -13,17 +13,188 @@
+With the advent of the tutorial-notebooks repo, the requirement for performing substantive computations and/or service load has been eliminated from system-test.
+With the proposed data curation notebooks, that require specific data holdings can live elsewhere.
+To the extent that this is practical, system-test service notebooks should be written with a data discovery or data check step to see what data is available (eg. in terms of available catalogs, tap_schema could be queried first to make sure unavailable catalogs are not being requested).
+In practice this is may be an aspirational requirement.
+
+Branches
+--------
+
+**Before:** Mobu typically runs the prod branch of the notebooks (though this is configurable) with notebooks having to be cherry-picked from main to prod.
+
+**After:** End this madness.
+
+**Discussion:**
+
+The need to maintain two different branches has been eliminated with mobu's ability to easily be configured to run off different branches for cases where it is useful to have an "in-development" version deployed.
+Hence cherry-picking is just annoying with no particular benefit.
+
+Outputs
+-------
+
+**Before:** Notebook contributors need to remember to clear outputs before checking in new versions of the notebooks.
+
+**After:** Have this happen automatically (via pre-commit hook or similar), or at least raise a CI warning if there is output checked in.
+
+**Discussion:**
+
+Having the human remember to clear outputs before saving and checking in is error prone. Even if the notebook ends in clear_outputs(), it still implies it was run to the end before commit.
+Ideally something like https://github.com/srstevenson/nb-clean would be integrated in the development workflow.
+
+This may also be of use to other notebook repo maintainers.
+
+
+Write-Only
+----------
+
+**Before:** Notebooks are checked out write-only in nublado to avoid conflicts
+
+**After:** Tutorial-notebooks continue to be write-only (this has saved a lot of support headaches) but system-test notebooks can be read-write to allow for easier guided troubleshooting.
+
+Directories
+-----------
+
+**Before:** Mobu runs notebooks at the root level of repos [?]
+
+**After:** Indicate to mobu whether to run a subdirectory (through eg. a .run file)
+
+**Discussion:** It would be nice to have a top-level directory structure for multiple runable repos, particularly for the data curation notebooks.
+This is not terribly important.
+
+Summit
+------
+
+**Before:** Mobu does not run at the summit
+
+**After:** There are system-test notebooks that probe basic "telescope" functionality (eg communication with DDS)
+
+**Discussion:** These had better be passive, we don't want to move the telescope or anything.... We should check what if any protections there are for this, eg is there further authorisation required to perform certain tasks
+
+Timing
+------
+
+**Before:** We have no timing information related to whole-notebook runs or per-cell runs
```

Comment:
mobu collects all of that timing information and makes it available via its API. We just don't use it for anything and it's stored in an awkward and incredibly verbose format.

Reply:
Will clarify

Reply:
Addressed here: https://github.com/lsst-sqre/sqr-080/commit/971379ef9a866a7ad64804a2afca1c7991e5c5e6